### ISSUE TITLE
Changed EC number in other databases

### ISSUE THREAD

**Issue by FaribaRoshanzamir** [SysBioChalmers/Human-GEM#366]
### Description of the issue:
Protoporphyrin ferrochelatase enzyme (FECH) was classified as EC 4.99.1.1 but now it's classified as 4.98.1.1. This change is already updated in several databases including the Enzyme database, KEGG, MetaCyc. 
I came across this one accidentally, but I am wondering if there is any automatic way to do these checkings and update the model once in a while? 

#### Expected feature/value/output:
We can update this one manually, for now. If you @mihai-sysbio will update it on the Metabolic atlas, I think we should keep the old EC number as well. 

#### Reproducing these results:

**I hereby confirm that I have:**
- [ ] Tested my code on my own computer for running the model
- [ ] Done this analysis in the `main` branch of the repository
- [ ] Checked that a similar issue does not exist already

**Comment by mihai-sysbio** [comment]
> I came across this one accidentally, but I am wondering if there is any automatic way to do these checkings and update the model once in a while?

That's a very good question - I vaguely remember discussions about it, but I could not find any implementation of this functionality. Perhaps @JonathanRob, @haowang-bioinfo or even @pecholley remember something.

> If you @mihai-sysbio will update it on the Metabolic atlas

Metabolic Atlas strives to be a perfect mirror of the model content, so if a change on the website is expected, it should come from a change in the model.

> We can update this one manually, for now.

Please go ahead @FaribaRoshanzamir 👍🏻

**Comment by JonathanRob** [comment]
No, I'm not really aware of any automatic functionality that could be used to update this, unfortunately.

**Comment by haowang-bioinfo** [comment]
EC number changes but not frequently. @FaribaRoshanzamir please go ahead to update this if provided with evidence.

**Comment by mihai-sysbio** [comment]
GECKO 3 contains a function to `getECfromDatabases` via KEGG mapping. Perhaps this is a good opportunity to bulk update the EC codes, and also reduce the missing values by about ~1900 entries.

**Comment by edkerk** [comment]
Confirmed and updated. [EC 4.99.1.1](https://enzyme.expasy.org/EC/4.99.1.1) is a transferred entry pointing to [EC 4.98.1.1](https://enzyme.expasy.org/EC/4.98.1.1) (protoporphyrin ferrochelatase), following the IUBMB creation of sub-class 4.98 for enzymes that form carbon-metal bonds. [MAR01044](https://metabolicatlas.org/explore/Human-GEM/gem-browser/reaction/MAR01044) (Ferrochelatase, FECH) now carries 4.98.1.1.

I used the current code only rather than keeping both. The old number is a deleted entry, so retaining it would be a dead lookup for EC-based tools such as GECKO, and the reclassification history stays visible at ExPASy anyway.

**Comment by edkerk** [comment]
Following up on the specific FECH fix with the broader question about checking these automatically. I ran the model's existing EC numbers against the ExPASy ENZYME database (`enzyme.dat`), which flags every transferred and deleted entry, so this is straightforward to repeat whenever the model is updated.

| Category | Reactions |
|---|---|
| EC transferred to a new number, updated | 507 |
| Erroneous EC corrected (typo, spurious, or deleted with a clear replacement) | 4 |
| Ambiguous deleted or split EC, left for manual review | 9 |

Most of the transfers are the 2018 reclassifications: the ATPase transporters moving into the new translocase class (for example [3.6.3.44](https://enzyme.expasy.org/EC/3.6.3.44) to [7.6.2.2](https://enzyme.expasy.org/EC/7.6.2.2)) and many cytochrome P450s renumbered from 1.14.13.x to 1.14.14.x. Where one old EC had been split into several, the right successor was chosen from the reaction's KEGG and gene evidence, or the closely related successors were kept together.

The four corrected errors were `1.4.11.1` to `1.14.11.1` and `2.1.3.85` to `2.3.1.85` (both digit-transposition typos), a spurious `5.2.2.2` removed from phosphoglucomutase, and the deleted `3.1.2.15` updated to `3.4.19.12`. The nine left open are deleted entries without a clear single replacement, or splits into unrelated ECs, which need a per-reaction decision.
